### DISCUSSION TITLE
若依

### DISCUSSION THREAD

**Discussion by xiaonuan-luck** [MisakaTAT/Shiro#218] [Q&A]
为什么整合进若依后就没办法连接上了啊，配置里也允许匿名访问了

**Comment by MisakaTAT** [comment]
> 为什么整合进若依后就没办法连接上了啊，配置里也允许匿名访问了

可能需要等我五一回去之后看了，手边现在没电脑，是否能够提供一个整个ruoyi的最小demo？

**Comment by XM2510136957** [comment]
因为ruoyi的拦截器需要单独配置一下白名单
